content/applications/sales/rental/manage_deposits.rst

```diff
@@ -0,0 +1,108 @@
+===============
+Manage deposits
+===============
+
+Requiring fixed deposits is common in many rental scenarios; such as collecting security deposits.
+
+This document covers the standard options for collecting and refunding deposits within the
+**Rentals** app:
+
+#. :ref:`rental/manage_deposits/config-optional-product`: Users are prompted to add the deposit to
+   the sales order upon selecting the rental product.
+#. :ref:`rental/manage_deposits/config-combo-product`: The deposit is automatically added to the
+   sales order upon selecting the rental product.
+#. :ref:`rental/manage_deposits/refund`: Reimburse deposits with a credit note.
+
+.. _rental/manage_deposits/config-service-product:
+
+Configuration
+-------------
+
+First create a *deposit service product* for each rental product that requires a deposit.
+
+To do so, go to :menuselection:`Rental --> Products` and create a :guilabel:`New` product.
+
+On the product form, give the product a name that indicates it is a deposit.
+
+.. example::
+   If this deposit service product is intended for the *Digital Camera* rental product, the name may
+   be entered as `Digital Camera Deposit`.
+
+Next, set the :guilabel:`Product Type` to :guilabel:`Service` and assign a :guilabel:`Sales Price`
+with the amount to collect for the deposit. Choose whether or not to include :guilabel:`Sales Taxes`
+for this deposit service product.
+
+Be sure that the :guilabel:`Rental` checkbox is ticked at the top of the product form.
```

Comment:
Potentially removing this line, however, by not configuring the deposit service product as **Rental**, it becomes unavailable for the combo product workflow. TBD.
```suggestion
```